### PR TITLE
[Fix] Transform new documentation site tables from React to Markdown

### PR DESCRIPTION
**Pull request by ronijaakkola**
## Description
This PR transforms new documentation site tables that were implemented in React to Markdown. This mostly relates to token documentation but also the Button documentation page.

## Related Issue
https://helsinkisolutionoffice.atlassian.net/browse/HDS-1202

## How Has This Been Tested?
Tested by running the documentation site locally.

### CODE REVIEW

File: new-site/src/components/examples/FocusExample.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FocusExample = ({ color, background, style = {}, children }) => (
```
Comment (minevala) [inline new-site/src/components/examples/FocusExample.js:4]: Should we add the aria-labels and roles to this FocusExample element and for the ColourExample element (if missing)?
https://helsinkisolutionoffice.atlassian.net/wiki/spaces/DD/pages/7809761295/25.2.2022+saavutettavuustunti

**Review by minevala — COMMENTED** [review]
Nice, simplifies the code.
One accessibility related comment

File: new-site/src/docs/elements/components/buttons/code.mdx [hunk 2]
```diff
@@ -228,9 +222,23 @@ import Playground from '../../../../components/Playground';
...
 </Playground>
 
-
-<PropertyTable />
-
-Note! You can find the full list of properties in the <Link size="M" href="https://hds.hel.fi/storybook/react/?path=/docs/components-button--primary" external>React Storybook.</Link>
+### Packages
+| Package           | Included                      | Storybook link                                                                                                                      | Source link   |
+| ----------------- | ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| **HDS React**     | <IconCheckCircleFill /> Yes   | <Link size="M" external href="/storybook/react/?path=/story/components-button--primary">View in Storybook</Link>  | <Link size="M" external href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/react/src/components/button">View source</Link> |
```
Comment (VilleMiekkoja) [inline new-site/src/docs/elements/components/buttons/code.mdx:228]: Import for IconCheckCircleFill is missing. I can add it to global space.

- [ ] Add IconCheckCircleFill as global so we do not need to import it in every file.

**Review by VilleMiekkoja — APPROVED** [review]
Well done 👍

File: new-site/src/components/examples/FocusExample.js (hunk 1 above)
Comment (minevala) [reply new-site/src/components/examples/FocusExample.js:4]: - [x] add missing aria-labels and role

**Review by minevala — APPROVED** [review]

**Pull request merged**